example/multi_threaded_inference/multi_threaded_inference.cc

```diff
@@ -0,0 +1,351 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * Copyright (c) 2017 by Contributors
+ * \file multi_threaded_inference.cc
+ * \brief Multi Threaded inference example with CachedOp
+*/
+
+#include <cstdio>
+#include <cstdlib>
+#include <iostream>
+#include <fstream>
+#include <thread>
+#include <iomanip>
+#include <chrono>
+#include <mxnet/ndarray.h>
+#include <opencv2/opencv.hpp>
+#include <mxnet/c_predict_api.h>
+#include "mxnet-cpp/MxNetCpp.h"
+
+const mx_float DEFAULT_MEAN = 117.0;
```

Comment:
Let's just use float. I don't think we will need this typedef any more.

Reply:
changed to float